packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -28,7 +29,7 @@ const DEFAULT_COMMAND_TTL = new Map([
 module.exports = (
   log,
   db,
-  oauthdb,
+  oauth,
```

Comment:
Is `oauth` the Best Practice (r) name for the raw db?

Reply:
I just copied this from other routes that were using it; `oauthdb` seems better but risks confusion with the other `oauthdb`.